apps/etterlatte-saksbehandling-ui/client/src/components/klage/formkrav/KlageFormkravRedigering.tsx

```diff
@@ -207,53 +217,86 @@ export function KlageFormkravRedigering() {
           <JaNeiRadiogruppe name="erFormkraveneOppfylt" control={control} legend="Er formkravene til klagen oppfylt?" />
 
           <VurderingWrapper>
-            <Textarea {...register('begrunnelse')} label="Totalvurdering (valgfritt)" />
+            <Textarea {...register('begrunnelse')} label="Totalvurdering (valgfritt)" readOnly={!redigerModus} />
           </VurderingWrapper>
+          {redigerModus ? (
+            <FlexRow justify="center">
+              <Button type="submit" loading={isPending(lagreFormkravStatus)}>
+                Lagre vurdering av formkrav
+              </Button>
+            </FlexRow>
+          ) : (
+            <>
+              <FlexRow $spacing>
+                <Button onClick={() => setRedigerModus(true)} variant="secondary">
+                  Endre vurdering
+                </Button>
+              </FlexRow>
+              <VurderingWrapper>
+                {!kanVurdereUtfall(klage) && (
```

Comment:
Jeg skjønner ikke helt når denne skal dukke opp? I alle fall med implementasjonen nå har vi en ikke-redigeringsmodus klage der `kanVurdereUtfall(klage) === false`, men hvordan er det mulig å lagre formkrav der man ikke har svart på spørsmålet `klage?.formkrav?.formkrav.erFormkraveneOppfylt`?

Reply:
A widget arrives precisely when he means to! Neida, skal se litt på det :)

Reply:
Skjønner ikke helt dette. Du skal vel ikke lagre hvis ikke alle spm er besvart... La også til en commit med en liten justering.

Reply:
var jeg som blingset litt på forskjellen mellom klagefrist og formkrav her!